nixos/doc/manual/development/meta-attributes.section.md

```diff
@@ -23,15 +23,15 @@ file.
 
   meta = {
     maintainers = with lib.maintainers; [ ericsagnes ];
-    doc = ./default.xml;
+    doc = ./default.md;
     buildDocsInSandbox = true;
   };
 }
 ```
 
 -   `maintainers` contains a list of the module maintainers.
 
--   `doc` points to a valid DocBook file containing the module
+-   `doc` points to a valid CommonMark file containing the module
```

Comment:
Maybe we can point to https://nixos.org/manual/nixpkgs/unstable/#sec-contributing-markup here?